source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```diff
@@ -0,0 +1,84 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "7.0"
+     csfle: true
```

Comment:
Indentation appears off here. `csfle` and `minServerVersion` are both child fields of the [`runOnRequirement` object](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.md#runonrequirement).